lib/elixir/lib/module.ex

```diff
@@ -861,6 +861,22 @@ defmodule Module do
     end, tuples)
   end
 
+  @spec make_overridable(module, module) :: :ok
+  def make_overridable(module, behaviour) when is_atom(module) and is_atom(behaviour) do
+    function_definitions = :ets.match(defs_table_for(module), {{:def, :"$1"}, :def, :"_", :"_", :"_", :"_"})
```

Comment:
def and defmacro can also be overridden. I would suggest removing the second `:def` so we get all kinds.